packages/expo-cli/e2e/__tests__/export-test.ts

```diff
@@ -52,6 +55,66 @@ it('exports the project for a self-hosted production deployment', async () => {
   expect(distFiles).toMatchSnapshot();
 });
 
+it('should export hbc bundle if running with hermes', async () => {
+  jest.setTimeout(5 * 60e3);
+  const tempDir = temporary.directory();
+  try {
+    // Require sdk 40+ to use @expo/dev-server for generating hbc
+    const projectRoot = await createMinimalProjectAsync(
+      tempDir,
+      'export-test-app',
+      {
+        sdkVersion: '41.0.0',
+      },
+      {
+        expo: '41.0.1',
+        'react-native': '0.63.2',
+      }
+    );
+    const androidRoot = path.join(projectRoot, 'android');
+    await fs.ensureDir(androidRoot);
+    await fs.writeFile(path.join(androidRoot, 'gradle.properties'), 'JS_RUNTIME=hermes');
+    await fs.writeFile(
+      path.join(projectRoot, 'babel.config.js'),
+      `
+module.exports = function(api) {
+  api.cache(true);
+  return {
+    presets: ['babel-preset-expo'],
+  };
+};
+`
+    );
+
+    const dotExpoHomeDirectory = path.join(projectRoot, '../.expo');
+    await runAsync(
```

Comment:
Would you mind adding a test for:
` ['expo', 'export', '--output-dir', inputDir, '--experimental-bundle']`
so we can ensure it works with eas publish? 

https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/project/publish.ts#L157

Reply:
would you mind if i testing this locally?
b/c the --experimental-bundle output format might change in near future, for those we talked about the asset mimetype or keeping original js bundle.

yet another reason, the e2e test takes longer time in CI.

btw
from my previous test, the command works fine and generated bundle is hbc.

Reply:
yes local is fine. 
Agreed it will change.

Just wanted to be sure you were aware of this path as eas cli relies on it.

Reply:
thanks. will make sure --experimental-bundle not broken.